packages/typegpu/tests/tgsl/wgslGenerator.test.ts

```diff
@@ -958,11 +958,25 @@ describe('wgslGenerator', () => {
     });
 
     expect(() => parseResolved({ testFn })).toThrowErrorMatchingInlineSnapshot(`
-[Error: Resolution of the following tree failed: 
+[Error: Resolution of the following tree failed:
 - <root>
 - fn:testFn
-- vec4f: Resolution of the following tree failed: 
+- vec4f: Resolution of the following tree failed:
 - vec4f: Cannot convert argument of type 'array' to 'f32' for function vec4f]
 `);
   });
+
+  it('generates correct code for pointer value assignment', () => {
+    const increment = tgpu.fn([d.ptrFn(d.f32)])((val) => {
+      // biome-ignore  lint/style/noParameterAssign: <go away>
```

Comment:
We don't actually need these brackets, they're just in the original message to note that it's a placeholder
```suggestion
      // biome-ignore  lint/style/noParameterAssign: go away
```